.github/workflows/submit-kedro-pipeline.yml

```
@@ -134,9 +134,9 @@ jobs:
         run: |
           echo "Submit the kedro pipeline"
           if [ "${{ env.BUMP_TYPE }}" == "minor" ]; then
-            RELEASE_PIPELINE='kg_release'
+            RELEASE_PIPELINE='kg_release_and_matrix_run'
           else
-            RELEASE_PIPELINE='kg_release_patch'
+            RELEASE_PIPELINE='kg_patch_and_matrix_run'
```

Trying to keep it consistent :) 
```suggestion
            RELEASE_PIPELINE='kg_release_patch_and_matrix_run'
```